### PR TITLE
Running Low Warning

### LINKED ISSUE #89: As a user, I can receive warnings when certain food items are running low after I remove a food item
so that I do not have to personally check for food quantity

### PR DESCRIPTION
**Pull request by kwokyto**

### CODE REVIEW

File: src/main/java/seedu/fridgefriend/command/RemoveCommand.java [hunk 1]
```diff
@@ -107,5 +89,18 @@ public String getMessagePrintedToUser() {
...
+    
+    /**
+     * Checks if the amount of food left for that category is insufficient.
+     * Appends a warning message to the user if true.
+     */
+    private void checkRunningOut() {
+        FoodCategory foodCategory = foodToBeEditted.getCategory();
+        if (fridge.isRunningOut(foodCategory)) {
+            int totalQuantity = fridge.getTotalQuantity(foodCategory);
+            this.runningOutMessage = "\nWARNING! " + foodCategory.toString() + " is running low on food!\nTotal "
```
Comment (Vinci-Hu) [inline src/main/java/seedu/fridgefriend/command/RemoveCommand.java]: I suggest You are running low on CATEGORY

**Review by Vinci-Hu — APPROVED** [review]
LGTM except the output message. Trivial though.

File: src/main/java/seedu/fridgefriend/command/RemoveCommand.java (hunk 1 above)
Comment (kwokyto) [reply src/main/java/seedu/fridgefriend/command/RemoveCommand.java]: changes made

**Review by SimJJ96 — APPROVED** [review]
LGTM

**Review by Leeyp — APPROVED** [review]
LGTM

**Pull request merged**